resallocserver/manager.py

```diff
@@ -246,15 +247,27 @@ def job(self):
             resource.state = RState.ENDED if output['status'] else RState.UP
             resource.data = output['stdout']
             tags = []
-            if type(self.pool.tags) != type([]):
+            if not isinstance(self.pool.tags, list):
                 msg = "Pool {pool} has set 'tags' set, but that's not an array"\
                         .format(pool=self.name)
                 warnings.warn(msg)
             else:
                 for tag in self.pool.tags:
+                    tag_name = None
+                    tag_priority = 0
+                    if isinstance(tag, str):
+                        # older format
+                        tag_name = tag
+                    elif isinstance(tag, dict):
+                        tag_name = tag['name']
+                        tag_priority = tag.get('priority', 0)
+                    else:
+                        assert False
```

Comment:
I would probably `raise` some custom error.

Reply:
This would be a game-over situation .. and I'm a bit lazy to invent an exception that is never going to be handled ...